tools/sgen/input/FluentUIStyle.yml

```diff
@@ -44,6 +44,23 @@ Colors:
     - brandSelected: "FluentUIColor(light: $Colors.Brand.shade20, dark: $Colors.Brand.shade20)"
     - brandDisabled: "FluentUIColor(light: NamedColor(grey74), dark: NamedColor(grey36))"
 
+  Notification:
+    - primaryToastForeground: "FluentUIColor(light: $Colors.Brand.shade10, dark: $Colors.Brand.shade30)"
+    - primaryBarBackground: "FluentUIColor(light: $Colors.Brand.tint40, dark: $Colors.Brand.tint10)"
+    - primaryBarForeground: "FluentUIColor(light: $Colors.Brand.shade20, dark: NamedColor(black))"
+    - primaryOutlineBarBackground: "FluentUIColor(light: $Colors.Brand.primary, dark: #F7F7F7)"
+    - subtleBarBackground: "FluentUIColor(light: #FFFFFF, dark: #393939)"
+    - neutralBarBackground: "FluentUIColor(light: #DFDFDF, dark: #393939)"
+    - neutralToastBackground: "FluentUIColor(light: #F7F7F7, dark: #393939)"
+    - warningToastBackground: "FluentUIColor(light: #FFFBD6, dark: #4C4400)"
+    - dangerToastBackground: "FluentUIColor(light: #FDF6F6, dark: #3F1011)"
```

Comment:
Colors from Waring and Danger styles will probably need to added to the asset catalog. I believe they're some shared colors (red and yellow). Ping me for the link to the figma file where the shared colors are defined.